src/connection.js

```diff
@@ -78,7 +78,7 @@ class Connection extends EventEmitter {
         enableAnsiWarnings: true,
         enableArithAbort: false,
         enableConcatNullYieldsNull: true,
-        enableCursorCloseOnCommit: undefined,
+        enableCursorCloseOnCommit: false,
```

Comment:
You need to revert the changes made for `enableCursorCloseOnCommit` 😅 More details in #643 & #653

Reply:
:stuck_out_tongue_closed_eyes: Sure